### ISSUE TITLE
Poor performance observed with models trained using the Training notebook

### ISSUE THREAD

**Issue by chenxd1996** [shyamsn97/mario-gpt#25]
Hello,

I have been using the Training notebook provided in this repository to train my model, and I've encountered an issue where the performance of the trained model is significantly subpar.

![image](https://github.com/shyamsn97/mario-gpt/assets/15684585/bbe3070d-2920-4ec3-8647-20b222719117)

**Comment by shyamsn97** [comment]
Hey! Can you share your training parameters? Like number of epochs, learning rate, etc. I think by default the notebook doesn’t train for long, so that could be an issue

**Comment by chenxd1996** [comment]
> Hey! Can you share your training parameters? Like number of epochs, learning rate, etc. I think by default the notebook doesn’t train for long, so that could be an issue


Oh, I see! I've been using the default training parameters. That might be the issue then. Could you please advise on the optimal settings for the training parameters for better performance?

**Comment by shyamsn97** [comment]
In the original work I trained for 50k iterations with a batch size of 4 (so the model ends up seeing 200,000 samples). Although I think you can get away with a lot less

**Comment by shyamsn97** [comment]
Closed this, feel free to open it if you have more questions!
